sdk/core/azure-core/inc/azure/core/diagnostics/logger.hpp

```diff
@@ -54,7 +54,17 @@ namespace Azure { namespace Core { namespace Diagnostics {
     static void SetLevel(Level level);
 
   private:
+    /**
+     * @brief An instance of `%Logger` class cannot be created.
+     *
```

Comment:
Same here, @vhvb1989 is this extra new line required still?
I thought we didn't need it anymore. @antkmsft  please confirm and check if we are consistent elsewhere (it was modified recently). I think removing this extra line is necessary to get rid of the "more..." link.

Reply:
@vhvb1989, should we ever have them, or only sometimes, or never?

Reply:
Maybe will be in a follow up PR, depending on the answer

Reply:
We want it for docs inside a class, like properties and methods.

We don't want it for documentation out of classes, like struct/class/namespace/global stuff

Reply:
Victor: "We want it for docs inside a class, like properties and methods.

We don't want it for documentation out of classes, like struct/class/namespace/global stuff"